libs/ledger-live-common/src/families/cardano/specs.ts

```diff
@@ -163,6 +164,67 @@ const cardano: AppSpec<Transaction> = {
         );
       },
     },
+    {
```

Comment:
Any chance to add a `test` field in each mutation, similarly to the current ones? In case there are additional expectations that could be tested.

Reply:
yes I'll add. probably I can only check for op type will do that 👍